ui/admin/tests/acceptance/auth-methods/selectors.js

```diff
@@ -90,3 +90,10 @@ export const TABLE_FIRST_ROW_ACTION_DROPDOWN =
   'tbody tr:first-child td:last-child button';
 export const TABLE_FIRST_ROW_ACTION_FIRST_ITEM =
   'tbody tr:first-child td:last-child ul li:first-child button';
+
+export const SEARCH_BAR_INPUT = '[type=search]';
+
+export const TABLE_ACTION_DROPDOWN = (authMethodId) =>
+  `tbody [data-test-auth-methods-table-row="${authMethodId}"] .hds-table__td:last-child .hds-dropdown button`;
+export const TABLE_ACTION_DROPDOWN_MAKE_PRIMARY = (authMethodId) =>
+  `tbody tr[data-test-auth-methods-table-row="${authMethodId}"] td:last-child ul li:first-child button`;
```

Comment:
Nice work. Wondering why we are selecting on `td:last-child` and also selecting on `ul li:first-child button`? I pulled this down and reproduced the user flow. It behaves as expected when selecting only on `ul li:first-child button`. Curious since the list only has 1 item.

Reply:
The `td:last-child` makes sure it is the last cell of the table.

It is true, deleting the `td:last-child` part behaves as expected today, but in this case, I wanted to make explicit the selection of the action dropdown, which is at the last cell.